xml/containers-docker-installation.xml

```diff
@@ -0,0 +1,266 @@
+    <para>
+     This automatically starts <systemitem>docker.socket</systemitem>.
+    </para>
+   </step>
+  </procedure>
+
+  <para>
+   The &docker; daemon listens on a local socket accessible only by the
+   <systemitem class="username">root</systemitem> user and by the members of
+   the <systemitem class="groupname">docker</systemitem> group. The
+   <systemitem class="groupname">docker</systemitem> group is automatically
+   created during package installation.
+  </para>
+
+  <para>
+   To allow a certain user to connect to the local &docker; daemon, use the
+   following command:
+  </para>
+
+<screen>&prompt.sudo;/usr/sbin/usermod -aG docker <replaceable>USERNAME</replaceable></screen>
+
+  <para>
+   This allows the user to communicate with the local &docker; daemon.
+  </para>
+ </sect1>
+ <sect1 xml:id="sec-docker-setup-net">
+  <title>Configuring the Network</title>
+
+  <para>
+   To give the containers access the external network, enable the <option>ipv4
+   ip_forward</option> rule.
+  </para>
+
+  <sect2 xml:id="sec-docker-setup-net-forward">
+   <title>Enabling IPv4 Forwarding</title>
+   <para>
+    This can be done using &yast; by switching to <menuchoice>
+    <guimenu>System</guimenu> <guimenu>Network Settings</guimenu>
+    <guimenu>Routing</guimenu> </menuchoice> menu and ensuring <option>Enable
+    IPv4 Forwarding</option> is checked.
+   </para>
+   <para>
+    The option cannot be changed when networking is handled by the Network
+    Manager. In that case, edit the
+    <filename>/etc/sysconfig/SuSEfirewall2</filename> file manually to ensure
+    the <option>FW_ROUTE</option> flag is set to <option>yes</option>:
+   </para>
+<screen>FW_ROUTE="yes"</screen>
+  </sect2>
+
+  <sect2 xml:id="sec-docker-setup-net-power">
+   <title>Networking Limitations on the &power; Architecture</title>
+   <para>
+    Currently &docker; networking has two limitations on the &power;
+    architecture.
+   </para>
+   <para>
+    &productname; machines cannot run &deng; with the iptables support enabled.
+    This issue is expected to be solved in an kernel update. In the meantime,
+    the <package>docker</package> package for &power; has iptables support
+    disabled via a dedicated directive inside of
+    <filename>/etc/sysconfig/docker</filename>.
+   </para>
+   <para>
+    Because of this limitation, &docker; containers cannot have access to the
+    outer network. A possible workaround is to share the same network namespace
+    between the host and the containers. However, this reduces the isolation of
+    the containers.
+   </para>
+   <para>
+    The network namespace of the host can be shared on a per-container basis by
+    adding <option>--net=host</option> to the <option>docker run</option>
+    command.
+   </para>
+   <note>
+    <title>iptables Support on &productname;</title>
+    <para>
+     &productname; hosts are not affected by this limitation, but they may have
```

Comment:
I think &productname; may be wrong here. Should it be &x86; or something? As in, this only affects POWER?